app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -49,9 +51,25 @@ fun MessageCard(
     messageText: String,
     titleText: String? = null,
     buttonText: String? = null,
+    colors: MessageCardColors = MessageCardColors.buildMessageCardColors(),
```

Comment:
```suggestion
    messageColors: MessageCardColors = MessageCardColors.buildMessageCardColors(),
```
I was thinking `colors` might be too generic here.

Reply:
This is not a strongly held opinion.

Reply:
I think this makes sense